library/servers/servers.h

```diff
@@ -34,6 +35,7 @@ struct proc_reply {
           CMD_OK,
           CMD_ERROR
     } type;
+    int pid;
```

Comment:
As mentioned earlier, if we remove the loop in `sys_shell.c`, we could further remove the `pid` here.

Reply:
The first commit removes `pid`.

The third commit adds back `pid` to allow for a child process to receive its parent's process id through the `PROC_PARENT` request to `GPID_PROCESS`. The comment below the `sys_shell` loop goes further into all the changes made within the third commit.